recipes/dataportal/v0.2.3/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: dataportal
+  version: v0.2.3
+
+source:
+  git_url: https://github.com/NSLS-II/dataportal
+  git_rev: v0.2.3
+
+build:
+  number: 1
+
+requirements:
+  build:
+    - python
+    - numpy
+    - six
+
+  run:
+    - python
+    - pandas
+    - pyyaml
+    - pims
+    - metadatastore >=0.2.0
+    - filestore >=0.2.0
+    - channelarchiver
+    - six
+    - humanize
```

Comment:
can you remove humanize? afaict it is not used anywhere in dataportal at the v0.2.3 tag